app/assets/stylesheets/stories.scss

```diff
@@ -84,3 +84,17 @@
     grid-area: submit;
   }
 }
+
+.popup {
+  display: none;
+  position: absolute;
+  top: 5px;
+  left: 30px;
+  background: #d9f1f1;
+  height: 30px;
+  width: 140px;
+  border: 1px solid #9b054d;
+  padding: 5px;
+  align-items: center;
```

Comment:
I though `align-items` and `justify-content` only work in flexboxes?

Reply:
I think they can work with grid too, but you are right, I had this leftover.